servicetalk-examples/grpc/compression/build.gradle

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright © 2019, 2021 Apple Inc. and the ServiceTalk project authors
```

Comment:
New files should have only the current year

Reply:
Since it is a copy of the hello world file it is a derived work and probably should retain the original copyright. Consider that if it had someone else's copyright on it and we copied and altered it for the compression it would not be different enough. If there is a standard for "new work" vs "derived work" I will be happy to follow that. It matters less in this case since there is unlikely to be a claim of copyright infringement by Apple against itself, but something to keep in mind for files with others copyrights, even if the license allows us to copy and modify.

Reply:
Good point! @Scottmitch ptal in case we need a new policy for copyright headers when the file is copied